### PR TITLE
Auto 495 delete estimate

### PR DESCRIPTION
**Pull request by ShannyWorkiz**

### CODE REVIEW

File: cypress/page-objects/General/generalPage.ts [hunk 1]
```diff
@@ -231,7 +231,7 @@ export class GeneralPage {
     }
 
     clickAddNewJobPhp(): CreateJobPage {
-        cy.get('.main-cta-wrapper.big-cta').click();
```
Comment (IdoAdler2) [inline cypress/page-objects/General/generalPage.ts:234]: @ShannyWorkiz Good job

File: cypress/page-objects/Estimate/allEstimatesPage.ts [hunk 2]
```diff
@@ -47,4 +54,14 @@ export class AllEstimatesPage {
         let estimateLine = this.getSpecificEstimateLine(estimateId)
         estimateLine.find('.rt-td:nth-child(4)').click({force:true});
     }
+
+    deleteEstimatesByIds(estimatesIdsList: JQuery<HTMLElement>[], amountToDelete: string){
```
Comment (IdoAdler2) [inline cypress/page-objects/Estimate/allEstimatesPage.ts:58]: @ShannyWorkiz I think the right thing is that amountToDelete will be from number type (not string), and then when u will use, it inside the 'type' function' convert it to string (amountToDelete).toString();

File: cypress/tests/core1/estimates.spec.ts [hunk 3]
```diff
@@ -42,4 +42,35 @@ describe('Estimate tests', () => {
...
+    });
+
+    it.only('After deleting multiply estimates from estimates report those estimates will not appear in estimates report', () => {
+        let estimate1 = generalPage.createNewEstimateWithClientFromAddNew();
+        pageRouter.goToEstimatesPage();
+        let estimate2 = generalPage.createNewEstimateWithClientFromAddNew()
+        let allEstimates = pageRouter.goToEstimatesPage();
+        cy.get(estimate1.alias).then((estimate1Id) => {
+            cy.get(estimate2.alias).then((estimate2Id) => {
+                allEstimates.deleteEstimatesByIds([estimate1Id, estimate2Id], '2');
```
Comment (IdoAdler2) [inline cypress/tests/core1/estimates.spec.ts:70]: @ShannyWorkiz see the note I gave u inside the function. Need to pass here number type, not string type.

File: cypress/tests/core1/estimates.spec.ts [hunk 4]
```diff
@@ -42,4 +42,35 @@ describe('Estimate tests', () => {
...
+            pageRouter.goToEstimatesPage().goToSingleEstimate(estimateId.toString()).deleteEstimate();
+            pageRouter.goToEstimatesPage().validateEstimatesTableNotContainsEstimateId(estimateId);
+        });
+    });
+
+    it.only('After deleting estimate from estimates report estimate will not appear in estimates report', () => {
+        let estimate = generalPage.createNewEstimateWithClientFromAddNew();
+        let allEstimates = pageRouter.goToEstimatesPage();
+        cy.get(estimate.alias).then((estimateId) => {
+            allEstimates.deleteEstimatesByIds([estimateId], '1');
```
Comment (IdoAdler2) [inline cypress/tests/core1/estimates.spec.ts:58]: @ShannyWorkiz see the note I gave u inside the function. Need to pass here number type, not string type.

**Review by IdoAdler2 — COMMENTED** [review]
good job. one small fix + delete onlys

File: cypress/tests/core1/estimates.spec.ts (hunk 3 above)
Comment (ShannyWorkiz) [reply cypress/tests/core1/estimates.spec.ts]: @IdoAdler2 FIxed

File: cypress/tests/core1/estimates.spec.ts (hunk 4 above)
Comment (ShannyWorkiz) [reply cypress/tests/core1/estimates.spec.ts]: @IdoAdler2 Fixed

File: cypress/page-objects/Estimate/allEstimatesPage.ts (hunk 2 above)
Comment (ShannyWorkiz) [reply cypress/page-objects/Estimate/allEstimatesPage.ts]: @IdoAdler2 That's smart! 
Fixed!

**Review by IdoAdler2 — APPROVED** [review]

**Pull request merged**